tests/std/tests/P2443R1_views_chunk_by/test.cpp

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <forward_list>
+#include <ranges>
+#include <span>
+#include <string_view>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+constexpr auto equal_ranges = [](auto&& left, auto&& right) { return ranges::equal(left, right); };
+constexpr std::less<> pred{};
+using Pred = std::less<>;

```

Comment:
This test is already `using namespace std;`. Additionally, previously `Pred` was determined with `decltype` so it had to appear second. Now we can reverse the order to centralize the type.